src/styles/theme/palette.ts

```diff
@@ -35,4 +35,7 @@ export const PALETTE: PaletteOptions = {
     hover: '#83E1FF',
   },
   divider: '#000',
+  custom: {
+    colorAquaBlue: '#73ddff',
```

Comment:
thank you for making this color part of the theme palette + updating type and tests!

small question: could it be some semantic name instead? I don't have screenshots, is it used only for some "main" button at Authors list? is it like "another active button color"?

Reply:
have no idea how to name it properly. let's leave it colorAquaBlue for now